plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -115,14 +126,16 @@
 
 		}
 
-		$stock_status_options     = wc_get_product_stock_status_options();
-		$stock_status_count       = count( $stock_status_options );
-		$common_stock_status_args = array(
+		$stock_status_options = wc_get_product_stock_status_options();
+		$stock_status_count   = count( $stock_status_options );
+		$stock_status_args    = array(
 			'id'            => '_stock_status',
 			'value'         => $product_object->get_stock_status( 'edit' ),
 			'wrapper_class' => 'stock_status_field hide_if_variable hide_if_external hide_if_grouped',
 			'label'         => __( 'Stock status', 'woocommerce' ),
 			'options'       => $stock_status_options,
+			'desc_tip'      => true,
+			'description'   => __( 'Controls whether or not the product is listed as "in stock" or "out of stock" on the frontend.', 'woocommerce' ),
```

Comment:
These changes are not directly related to the PR. It would be good in the future to either split them out or, since this is admittedly a small change and closely related, just reference that the change was made in the PR description/comments, so it is clear to the reviewer!